src/app/src/util/embeddedMap.js

```diff
@@ -2,16 +2,15 @@ import sortBy from 'lodash/sortBy';
 
 import { OARFont } from './constants';
 
-const DEFAULT_WIDTH = '1000';
-const DEFAULT_HEIGHT = '800';
+export const DEFAULT_WIDTH = '1000';
+export const DEFAULT_HEIGHT = '800';
```

Comment:
This may be for a follow-up card, but do we want to set a minimum as well? The map is unusable at small sizes.

![Screen Shot 2021-05-26 at 11 43 35 AM](https://user-images.githubusercontent.com/1042475/119690358-9d72be00-be17-11eb-9e8e-3da66e456859.png)

Reply:
@caseycesari My thought here is that, since users will have direct visual feedback that it looks terrible if they set it at a small size, we can let them self-police their sizing.